crates/erg_common/traits.rs

```diff
@@ -568,7 +567,7 @@ pub trait Runnable: Sized + Default {
     }
     fn run(cfg: ErgConfig) {
         crossterm::terminal::enable_raw_mode().unwrap();
```

Comment:
This is not needed here as we only receive input on stdin.rs
Please move this to stdin.rs's input